File: Python/Cpp Ports/Array Ordering.py
def sorter(values):
    count = len(values)
    
    for i in range(count-1):
        for j in range(i+1, count):
            if values[i] > values[j]:
                values[i], values[j] = values[j], values[i]
    return values


count = int(input("Enter the number of elements you want to enter: "))

values = [float(input("Enter the value: ")) for _ in range(count)]

sequence = sorter(values)

order = input("Ascending or Descending: ")

if order == "decending" or order == "Descending":
    for i in range(count-1, -1, -1): print(sequence[i])
elif order == "ascending" or order == "Ascending":
    for i in range(count): print(sequence[i])
else:
    print("Invalid input")
    

File: C++/__Array Ordering.cpp
#include <iostream>
#include <string>
#include <cmath>

using namespace std;

int main() {
  int n,i,j,ad;

//-------------------------------------------------Input-----------------------------------------------

  cout << "Enter count of numbers to be ordered \n";
    cin >> n; cout<<"\n";
  cout<<"Enter 1 for Ascending Order"<<"\n"<<"     2 for Decending Order"<<"\n";
  cin>>ad;
  cout <<"\n"<< "Type the numbers:";

  if(n<0){n=-1*n;}
  float x[n],

  for (i=1; i<=n; i++){
    cout << "\n";
      cin >> x[i];
  }

//-----------------------------------------------Processing------------------------------------------------
  for (j=1; j<=n-1; j++){
    for (i=j+1; i<=n; i++){
      if (x[j] < x[i]){
        a=x[i]; b=x[j];
        x[j]=a; x[i]=b;
      }
    }
  }

//-------------------------------------------------OutPut-----------------------------------------------------
  if(ad==1){
    cout<<"\n"<<"\n"<<"Ascending order of Array is :";

    for (i=n; i>=1; i--){
      cout<<"\n"<<order[i];
    }
  }

  else if(ad==2){
    cout<<"\n"<<"\n"<<"Decending order of Array is :";

    for (i=1; i<=n; i++){cout<<"\n"<<order[i];}
  }

  else {
    cout<<"Invalid Order Command";
  }

  return 0;
}
